repo/fsrepo/fsrepo.go

```diff
@@ -562,6 +563,22 @@ func (r *FSRepo) Datastore() ds.ThreadSafeDatastore {
 	return d
 }
 
+// GetDiskUsage computes the disk space taken by the repo in bytes
+func (r *FSRepo) GetDiskUsage() (int64, error) {
```

Comment:
maybe we should call it, `GetStorageUsage` ? as it won't always be Disk (e.g. S3 repo / datastore)